internal/repository/prisma/dbsqlc/tickers.sql

```diff
@@ -242,7 +239,10 @@ failed_run_count_by_tenant AS (
     WHERE
         "status" = 'FAILED'
         AND (
-            "lastAlertedAt" IS NULL OR
+            (
+                "lastAlertedAt" IS NULL AND
+                workflowRun."finishedAt" >= NOW() - convert_duration_to_interval(active_tenant_alerts."maxFrequency")
```

Comment:
quick check, we're validating active_tenant_alerts."maxFrequency"? on input?

Reply:
yeah we validate it's a duration string